src/containers/App/App.test.js

```diff
@@ -60,5 +60,55 @@ describe('App', () => {
     await wrapper.instance().componentDidMount()
 
     expect(wrapper.instance().props.addMovies).toHaveBeenCalledWith(mockMovies)
-  })
-})
+  });
+
+  describe('mapStateToProps', () => {
+    it('should return the movies data from state', () => {
+
+      const mockState = {
+        movies: [
+          {
+            id: 1,
+            title: "Jumanji: The Next Level",
```

Comment:
Just to make it more realistic, can you put the user and ratings key value pairs in here?

Reply:
@spaceplesiosaur Thank you, I have made the changes